### PR TITLE
store/tikv: encode/decode keys for pd

### PR DESCRIPTION
**Pull request by disksing**
@shenli

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by disksing** [comment]
@zimulala PTAL

File: store/tikv/pd_codec.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+type codecPDClient struct {
+	pd.Client
+}
+
+func (c *codecPDClient) GetTS() (int64, int64, error) {
+	return c.Client.GetTS()
+}
+
+// GetRegion encodes the key before send requests to pd-server and decodes the
+// returned StartKey & EndKey from pd-server.
```
Comment (zimulala) [inline store/tikv/pd_codec.go]: & -> && ?

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
You can only implement `GetRegion` method in `codePDClient`.

**Comment by coocood** [comment]
LGTM

**Pull request merged**